analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -35,4 +71,192 @@ class Bundler : PackageManager() {
     }
 
     override fun command(workingDir: File) = "bundle"
+
+    override fun resolveDependencies(definitionFile: File): AnalyzerResult? {
+        val workingDir = definitionFile.parentFile
+        val vendorDir = File(workingDir, "vendor")
+        var tempVendorDir: File? = null
+
+        try {
+            if (vendorDir.isDirectory) {
+                val tempDir = createTempDir("analyzer", ".tmp", workingDir)
+                tempVendorDir = File(tempDir, "vendor")
+                log.warn { "'$vendorDir' already exists, temporarily moving it to '$tempVendorDir'." }
+                Files.move(vendorDir.toPath(), tempVendorDir.toPath(), StandardCopyOption.ATOMIC_MOVE)
+            }
+            val scopes = mutableSetOf<Scope>()
+            val packages = mutableSetOf<Package>()
+            val errors = mutableListOf<String>()
+            val vcsDir = VersionControlSystem.forDirectory(workingDir)
+
+            installDependencies(workingDir)
+
+            val (projectName, version, projectHomepageUrl, declaredLicenses) = parseProject(workingDir)
+            val groupedDeps = getDependencyGroups(workingDir)
+
+            for ((groupName, dependencyList) in groupedDeps) {
+                parseScope(workingDir, groupName, dependencyList, scopes, packages, errors)
+            }
+
+            val project = Project(
+                    id = Identifier(
+                            provider = javaClass.simpleName,
+                            namespace = "",
+                            name = projectName,
+                            version = version
+                    ),
+                    declaredLicenses = declaredLicenses.toSortedSet(),
+                    aliases = emptyList(),
+                    vcs = vcsDir?.getInfo(workingDir) ?: VcsInfo.EMPTY,
```

Comment:
I believe this should be our usual contruct:

    vcs = VcsInfo.EMPTY,
    vcsProcessed = processProjectVcs(workingDir),

Reply:
Yes, we fill only `vcsProcessed` if the data is coming from the working tree and not from the projects metadata.